### PR TITLE
Add a welcome post

### PR DESCRIPTION
**Pull request by skade**
It's not very wordy, we have many other words incoming.

### CODE REVIEW

File: _posts/2017-04-02-welcome.md [hunk 1]
```diff
@@ -5,4 +5,13 @@ author: Florian Gilcher
...
-Some blurb!
+Hi Rustaceans!
+
+The Rust Community team is happy to announce the Rust Community blog!
+
+This blog is both intended as our regular outlet for community topics, as well as a highlight community events - meetups, usergroups out of the order events.
+
+If you have a community topic you would like write about here, please [contact us](mailto:community-team@rust-lang.org), or just [open an issue](https://github.com/rust-community/blog.community.rs).
+
+In the meantime, we'll kick of the blog with a series of posts describing the rust community team
```
Comment (erickt) [inline _posts/2017-04-02-welcome.md]: This feels like there was supposed to be more to this sentence.

File: _posts/2017-04-02-welcome.md [hunk 2]
```diff
@@ -5,4 +5,13 @@ author: Florian Gilcher
...
 ---
 
-Some blurb!
+Hi Rustaceans!
+
+The Rust Community team is happy to announce the Rust Community blog!
+
+This blog is both intended as our regular outlet for community topics, as well as a highlight community events - meetups, usergroups out of the order events.
+
+If you have a community topic you would like write about here, please [contact us](mailto:community-team@rust-lang.org), or just [open an issue](https://github.com/rust-community/blog.community.rs).
```
Comment (erickt) [inline _posts/2017-04-02-welcome.md]: We should start brainstorming what kind of standards we want for our guest posts. I'll open a ticket for that.

File: _posts/2017-04-02-welcome.md [hunk 3]
```diff
@@ -5,4 +5,13 @@ author: Florian Gilcher
 description: "The Rust Community Blog is open for business!"
 ---
 
-Some blurb!
+Hi Rustaceans!
+
+The Rust Community team is happy to announce the Rust Community blog!
+
+This blog is both intended as our regular outlet for community topics, as well as a highlight community events - meetups, usergroups out of the order events.
```
Comment (erickt) [inline _posts/2017-04-02-welcome.md]: I think you are missing a comma after "usergroups".

**Comment by skade** [comment]
Reworked the first paragraph a little, clarified that we don't merge just about everything.

**Pull request merged**